templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,18 @@
 # Press Enter to send
 
-By default, Zulip sends your message when you press enter (or return)
+By default, Zulip sends your message when you press **enter** (or **return**)
 on your keyboard. If you want to turn this off, simply uncheck the
 **Press Enter to send** box below the compose field:
 
 ![Box checked](/static/images/help/enter-box-checked.png)
 
-You will now see a **Send** button below the compose field looking
-like this.
+You will now see a **Send** button below the compose field like this:
 
 ![Box unchecked](/static/images/help/enter-box-unchecked.png)
```

Comment:
You can crop the image to show only the enter box, the rest is extraneous info. (Also, there might be duplicated images of the press enter to send in `static/images/help` folder, you should try and see if there are any and remove them)